src/pageObject.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return 'Conduit logo';
+  }
 };
 
 class PageObject {
+  constructor(url, header) {
+    this.url = url;
+    this.header = new Header();
+  }
 
+  clickOnLogo() {
+    return `Click on the ${this.header.logo}`;
+  }
 };
 
 class ArticlePage extends PageObject {
+  constructor(url, header, slug) {
+    super(url, header);
+    this.slug = slug;
+  }
```

Comment:
According to the task the URL should be modified by adding the slug at the end. You need to add/update a line of code that modifies the url by adding the slug at the end.

Reply:
done